Makefile

```diff
@@ -1,6 +1,9 @@
 BUILD_CHANNEL?=local
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
 PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):${PATH}"
+GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
+TAG_VERSION?=$(shell git tag --points-at | sort -Vr | head -n1)
```

Comment:
This follows james's advice here: https://github.com/viamrobotics/rplidar/pull/36#discussion_r1189060802

Reply:
validated against RDK